x-pack/plugins/ml/server/routes/system.ts

```diff
@@ -11,20 +11,17 @@ import { RequestHandlerContext } from 'kibana/server';
 import { wrapError } from '../client/error_wrapper';
 import { mlLog } from '../client/log';
 import { privilegesProvider } from '../lib/check_privileges';
-import { isSecurityDisabled } from '../lib/security_utils';
 import { spacesUtilsProvider } from '../lib/spaces_utils';
-import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
-import { RouteInitialization } from '../new_platform/plugin';
+import { licensePreRoutingFactory } from './license_check_pre_routing_factory';
+import { LicenseCheckResult } from '../types';
 
 /**
  * System routes
  */
-export function systemRoutes({
-  router,
-  xpackMainPlugin,
-  spacesPlugin,
-  cloud,
-}: RouteInitialization) {
+export function systemRoutes({ getLicenseCheckResults, router, spacesPlugin, cloud }: any) {
+  let cachedLicenseCheckResult: LicenseCheckResult;
+  // const { isSecurityDisabled } = getLicenseCheckResults();
```

Comment:
commented code can be removed

Reply:
609be5c4c218d1feed4b249158c438ce7c30f99d